### PR TITLE
[13.0] Wishlist: support item ordering

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

**Comment by simahawk** [comment]
@lmignon can you review this?

**Review by i-vyshnevska — APPROVED** [review]

File: shopinvader_wishlist/services/wishlist.py [hunk 1]
```diff
@@ -253,36 +261,49 @@ def _add_item(self, record, params):
...
-            existing.quantity += params["quantity"]
+            if update_params.get("quantity"):
+                update_params["quantity"] += existing.quantity
+            existing.update(update_params)
         else:
             self.env["product.set.line"].create(
-                self._prepare_item(params, record)
+                self._prepare_item(record, params)
             )
+        # TODO: WTF?? This does not work here, must be called in `_to_json_one`???
```
Comment (lmignon) [inline shopinvader_wishlist/services/wishlist.py:286]: @simahawk Do you still have an issue here?

Comment (lmignon) [reply shopinvader_wishlist/services/wishlist.py:286]: OK the flush is done into _to_json_one...

**Review by lmignon — APPROVED** [review]
Thank you @simahawk LGTM (code only)

**Pull request merged**